agent/grpc-external/services/peerstream/stream_resources.go

```diff
@@ -213,16 +233,12 @@ func (s *Server) HandleStream(streamReq HandleStreamRequest) error {
 			ResourceURL: resourceURL,
 			PeerID:      streamReq.RemoteID,
 		})
-		logTraceSend(logger, sub)
-
-		if err := streamReq.Stream.Send(sub); err != nil {
+		if err := streamSend(sub); err != nil {
 			if err == io.EOF {
 				logger.Info("stream ended by peer")
-				status.TrackReceiveError(err.Error())
```

Comment:
I think this was a mistake to use `status.TrackReceiveError` here instead of `status.trackSendError`